lsp-sqls.el

```diff
@@ -0,0 +1,76 @@
+;;; lsp-sqls.el --- SQL Client settings -*- lexical-binding: t; -*-
+
+;; Copyright (C) 2020 Shunya Ishii
+
+;; Author: Shunya Ishii
+;; Keywords: sql lsp
+
+;; This program is free software; you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published by
+;; the Free Software Foundation, either version 3 of the License, or
+;; (at your option) any later version.
+
+;; This program is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+
+;; You should have received a copy of the GNU General Public License
+;; along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+;;; Commentary:
+
+;; LSP client for SQL
+
+;;; Code:
+
+(require 'lsp-mode)
+(require 'json)
+
+(defgroup lsp-sqls nil
+  "LSP support for SQL, using sqls"
+  :group 'lsp-mode
+  :link '(url-link "https://github.com/lighttiger2505/sqls")
+  :package-version `(lsp-mode . "6.4"))
+
+(defcustom lsp-sqls-server "sqls"
+  "Path to the `sqls` binary."
+  :group 'lsp-sqls
+  :risky t
+  :type 'file
+  :package-version `(lsp-mode . "6.4"))
+
+(defcustom lsp-sqls-workspace-config-path "workspace"
```

Comment:
Instead of allowing any string, you can use as a choice like [here](https://github.com/emacs-lsp/lsp-mode/blob/31fe3c077fbb61d42e48108c2d02ca561f57c7be/lsp-vetur.el#L57-L59)